it/pom.xml

```diff
@@ -153,5 +153,15 @@
       <artifactId>JUnitParams</artifactId>
       <scope>test</scope>
     </dependency>
+    <dependency>
+      <groupId>com.google.apis</groupId>
+      <artifactId>google-api-services-cloudsearch</artifactId>
+      <version>v1-rev20230214-2.0.0</version>
+    </dependency>
+    <dependency>
+      <groupId>com.google.api-client</groupId>
+      <artifactId>google-api-client</artifactId>
+      <version>1.35.2</version>
```

Comment:
how did we decide this version?

Reply:
Good question, I tried to make this 2.2.0 but this was breaking the identity integration tests. I had to settle for this version only since fixing the issue could take more time.